Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -114,13 +111,7 @@ struct NimbusFlaggableFeature: HasNimbusSearchBar {
         }
 
         let nimbusValue = nimbusLayer.checkNimbusConfigFor(featureID)
-
-        switch featureID {
-        case .pocket:
-            return nimbusValue && PocketProvider.islocaleSupported(Locale.current.identifier)
-        default:
-            return nimbusValue
-        }
+        return nimbusValue
```

Comment:
I would argue we should just `return nimbusLayer.checkNimbusConfigFor(featureID)` if we no longer switch at all, because then there's no need to assign.